Classes/Console/Mvc/Cli/Symfony/Descriptor/TextDescriptor.php

```diff
@@ -173,17 +173,8 @@ protected function describeApplication(Application $application, array $options
             $this->writeText("\n");
 
             $commands = $description->getCommands();
-
-            if ($application instanceof \Helhum\Typo3Console\Mvc\Cli\Symfony\Application) {
-                $showUnavailable = $options['show_unavailable'] ?? false;
-                $commands = array_filter(
-                    $commands,
-                    function ($command) use ($application, $showUnavailable) {
-                        return $showUnavailable || $application->isCommandAvailable($command);
```

Comment:
Technically I get the same behavior if I keep this and only drop the `$showUnavailable` part, so I'm not sure if we still need this.